### PR TITLE
[blog] add new blog, saud-ansari, sae-baja

### PR DESCRIPTION
**Pull request by bitakshat**
feature [blog] Saud Ansari, Sae-baja, up and ready to merge

### CODE REVIEW

File: content/posts/saud-ansari/sae-baja/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+cover: "./banner.jpg"
+images:
+    - banner:
+        url: "https://unsplash.com/photos/7E6QksW4PUY"
+        feature: ./banner.jpg
+        credit:
+            info: "Image taken from unsplash.com"
+            link: https://unsplash.com/@suddenpanic
+
+    - two buggies:
```
Comment (pandevim) [inline content/posts/saud-ansari/sae-baja/index.md:17]: we would use underscore here not space

File: content/posts/saud-ansari/sae-baja/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        url: "https://unsplash.com/photos/hvSTh6aTd28"
+        feature: "two-buggies"
+        credit: 
+            info: Image taken from unsplash.com
+            link: "https://unsplash.com/@patricktkindt"
+
+date: "2021-04-19-13T00:27:00.00Z"
```
Comment (pandevim) [inline content/posts/saud-ansari/sae-baja/index.md]: incorrect date format, you cannot write April

File: content/posts/saud-ansari/sae-baja/index.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+lastupdated: "2021-07-29T12:16:46.00Z"
+
+categories:
```
Comment (pandevim) [inline content/posts/saud-ansari/sae-baja/index.md:27]: Blog belongs to only one category and multiple tags

File: content/posts/saud-ansari/sae-baja/index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+categories: 
+    - Automobile
+    - Technology
+    - Science
+tags:
+    - Vehicles
+    - Competition
+---
+
+![](./two-buggies.jpg)
```
Comment (pandevim) [inline content/posts/saud-ansari/sae-baja/index.md:36]: about the image inside []

File: content/posts/saud-ansari/sae-baja/index.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+The event ends with the final day 4 hour endurance race, the positions of the grid are decided by how much score a team has in their dynamic events. The team with most laps in the 4 hour long race is crowned the winner of the endurance race however the overall winner is decided by combining all the points from the Static and Dynamic events.
+
+---
```
Comment (pandevim) [inline content/posts/saud-ansari/sae-baja/index.md:45]: not trailing line after the blog is required

**Review by pandevim — CHANGES_REQUESTED** [review]

File: content/posts/saud-ansari/sae-baja/index.md (hunk 1 above)
Comment (bitakshat) [reply content/posts/saud-ansari/sae-baja/index.md:17]: like this?
info: image_taken_from_unsplash.com

File: content/posts/saud-ansari/sae-baja/index.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+    - two buggies:
+        url: "https://unsplash.com/photos/hvSTh6aTd28"
+        feature: "two-buggies"
+        credit: 
+            info: Image taken from unsplash.com
+            link: "https://unsplash.com/@patricktkindt"
+
+date: "2021-04-19T00:27:00.185Z"
+lastupdated: "2021-07-29T12:16:46.00Z"
```
Comment (pandevim) [inline content/posts/saud-ansari/sae-baja/index.md:25]: time format is incorrect, date and lastupdated should be similar

File: content/posts/saud-ansari/sae-baja/index.md (hunk 1 above)
Comment (pandevim) [reply content/posts/saud-ansari/sae-baja/index.md:17]: yes

File: content/posts/saud-ansari/sae-baja/index.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+        feature: ./banner.jpg
+        credit:
+            info: "Image taken from unsplash.com"
+            link: https://unsplash.com/@suddenpanic
+
+    - two buggies:
+        url: "https://unsplash.com/photos/hvSTh6aTd28"
+        feature: "two-buggies"
+        credit: 
+            info: image_taken_from_unsplash.com
```
Comment (pandevim) [inline content/posts/saud-ansari/sae-baja/index.md:21]: this is wrong, i mean two_buggiee

**Pull request merged**